packages/wrangler/src/__tests__/sentry.test.ts

```diff
@@ -21,10 +22,13 @@ describe("Error Reporting", () => {
     jest.mock("@sentry/node");
     jest.spyOn(Sentry, "captureException");
     process.stdout.isTTY = true;
+    jest.mock("ci-info");
+    (ci.isCI as jest.Mocked<boolean>) = false;
```

Comment:
`ciCheck` function doesn't make sense to modify to handle this use case, directly mocking `ci-info` 
Alternatively we can make a simple wrapper function that is just returns a boolean from, `ci.isCI`